src/features/device-state/state-get-utils.ts

```diff
@@ -86,11 +127,9 @@ export const serviceInstallFromImage = (
 			(si) => si.service[0].id === id,
 		);
 	} else if ('service' in deviceOrFleet) {
-		return deviceOrFleet.service.find(
```

Comment:
This is also slightly missleading as the fn is called `serviceInstallFromImage` but can return a service (rather than a serviceInstall). I decided not to change the function name for now as it would be a major and this is used on balena-api